user-interface/src/main/java/life/qbic/datamanager/views/register/UserRegistrationLayout.java

```diff
@@ -92,8 +98,13 @@ private void createFieldLayout() {
     fieldLayout = new VerticalLayout();
     createEmailField();
     createNameField();
+    createUsernameField();
     createPasswordField();
-    fieldLayout.add(fullName, email, password);
+    fieldLayout.add(fullName, email, username, password);
```

Comment:
NIT: I found it confusing that the username is inbetween the password and the email. I'd expected the string to login to be above the password 🤔